lib/debug/log.js

```diff
@@ -118,7 +118,7 @@ shaka.log.Level = {
 /**
  * @define {number} the maximum log level.
  */
-goog.define('shaka.log.MAX_LOG_LEVEL', 3);
+shaka.log.MAX_LOG_LEVEL = goog.define('shaka.log.MAX_LOG_LEVEL', 3);
```

Comment:
This misses another one in lib/debug/asserts.js.

But more broadly, I think I'd prefer to just stop using goog.define.  We already got rid of an instance of that in lib/player.js (using `@define` only), so we can do the same here and in asserts.js.

Reply:
The one in asserts is actually worse - goog.define was never intended to _change_ values that were already defined elsewhere.

What do you intend to replace it with?

Reply:
I'll be just assigning the variables directly without goog.define.  Just the `@define` annotation and the assignment.